MotionMark/resources/debug-runner/debug-runner.js

```diff
@@ -688,6 +688,7 @@ Utilities.extendObject(window.benchmarkController, {
             return false;
 
         setTimeout(function() {
+            benchmarkController.determineCanvasSize();
```

Comment:
I would move this out of the timeout callback, so that `startBenchmarkImmediatelyIfEncoded` looks more like `startBenchmark`.

Reply:
Updated below. PTAL.